project/controllers/incident_cntr.py

```diff
@@ -0,0 +1,45 @@
+from project.models import Incident, incidents
+
+
+class IncidentCntr:
+
+    def create_incident(self, incident):
+        """ 
+        method to create incident
+        """
+        new_incident = incident.to_json()
+        incidents.append(new_incident)
+        return incidents
+
+    def get_all_redflags(self):
+        """ 
+        gets all redflags
+        """
+        if len(incidents) > 0:
+            return incidents
+
+    def get_single_redflag(self, incident_id):
+        """ 
+        gets single redflags
+        """
+
+        for incident in incidents:
+            if incident['incident_id'] == incident_id:
+                return incident
+
+    def delete_single_redflag(self, incident_id):
+        """ 
+        deletes single  redflags
+        """
+        for incident in incidents:
```

Comment:
@3Nakajugo Where have you declared the incidents variable you are checking here?

Reply:
incidents is a list I imported from the modal